.github/workflows/codeql.yml

```
@@ -0,0 +1,91 @@
+name: "CodeQL Advanced"
+
+on:
+  push:
+    branches: [ "master", "codecarbon_v3_rc" ]
+  pull_request:
+    branches: [ "master", "codecarbon_v3_rc" ]
```

"codecarbon_v3_rc" no longer exist.
```suggestion
    branches: [ "master"]
  pull_request:
    branches: [ "master"]
```